diff --git a/node_modules/@cosmjs/tendermint-rpc/build/comet38/adaptor/responses.js b/node_modules/@cosmjs/tendermint-rpc/build/comet38/adaptor/responses.js
index e8b2dc3..b5c540a 100644
--- a/node_modules/@cosmjs/tendermint-rpc/build/comet38/adaptor/responses.js
+++ b/node_modules/@cosmjs/tendermint-rpc/build/comet38/adaptor/responses.js
@@ -72,7 +72,7 @@ function decodePubkey(data) {
     if ("Sum" in data) {
         // we don't need to check type because we're checking algorithm
         const [[algorithm, value]] = Object.entries(data.Sum.value);
-        (0, utils_1.assert)(algorithm === "ed25519" || algorithm === "secp256k1", `unknown pubkey type: ${algorithm}`);
+        (0, utils_1.assert)(algorithm === "ed25519" || algorithm === "secp256k1" || algorithm === "bn254", `unknown pubkey type: ${algorithm}`);
         return {
             algorithm,
             data: (0, encoding_1.fromBase64)((0, encodings_1.assertNotEmpty)(value)),
@@ -91,6 +91,11 @@ function decodePubkey(data) {
                     algorithm: "secp256k1",
                     data: (0, encoding_1.fromBase64)((0, encodings_1.assertNotEmpty)(data.value)),
                 };
+            case "tendermint/PubKeyBn254":
+                return {
+                    algorithm: "bn254",
+                    data: (0, encoding_1.fromBase64)((0, encodings_1.assertNotEmpty)(data.value)),
+                };
             default:
                 throw new Error(`unknown pubkey type: ${data.type}`);
         }
diff --git a/node_modules/@cosmjs/tendermint-rpc/build/tendermint37/adaptor/responses.js b/node_modules/@cosmjs/tendermint-rpc/build/tendermint37/adaptor/responses.js
index 5c9a406..0083a25 100644
--- a/node_modules/@cosmjs/tendermint-rpc/build/tendermint37/adaptor/responses.js
+++ b/node_modules/@cosmjs/tendermint-rpc/build/tendermint37/adaptor/responses.js
@@ -72,7 +72,7 @@ function decodePubkey(data) {
     if ("Sum" in data) {
         // we don't need to check type because we're checking algorithm
         const [[algorithm, value]] = Object.entries(data.Sum.value);
-        (0, utils_1.assert)(algorithm === "ed25519" || algorithm === "secp256k1", `unknown pubkey type: ${algorithm}`);
+        (0, utils_1.assert)(algorithm === "ed25519" || algorithm === "secp256k1" || algorithm === "bn254", `unknown pubkey type: ${algorithm}`);
         return {
             algorithm,
             data: (0, encoding_1.fromBase64)((0, encodings_1.assertNotEmpty)(value)),
@@ -91,6 +91,11 @@ function decodePubkey(data) {
                     algorithm: "secp256k1",
                     data: (0, encoding_1.fromBase64)((0, encodings_1.assertNotEmpty)(data.value)),
                 };
+            case "tendermint/PubKeyBn254":
+                return {
+                    algorithm: "bn254",
+                    data: (0, encoding_1.fromBase64)((0, encodings_1.assertNotEmpty)(data.value)),
+                };
             default:
                 throw new Error(`unknown pubkey type: ${data.type}`);
         }
