diff --git a/node_modules/@cosmjs/stargate/build/logs.js b/node_modules/@cosmjs/stargate/build/logs.js
index 38d7500..cdb1bc2 100644
--- a/node_modules/@cosmjs/stargate/build/logs.js
+++ b/node_modules/@cosmjs/stargate/build/logs.js
@@ -57,12 +57,13 @@ function parseLogs(input) {
 }
 exports.parseLogs = parseLogs;
 function parseRawLog(input = "[]") {
-    const logsToParse = JSON.parse(input).map(({ events }, i) => ({
-        msg_index: i,
-        events,
-        log: "",
-    }));
-    return parseLogs(logsToParse);
+    return input
+    // const logsToParse = JSON.parse(input).map(({ events }, i) => ({
+    //     msg_index: i,
+    //     events,
+    //     log: "",
+    // }));
+    // return parseLogs(logsToParse);
 }
 exports.parseRawLog = parseRawLog;
 /**
