diff --git a/rust-src/lib/rustlib/src/rust/library/std/Cargo.toml b/rust-src/lib/rustlib/src/rust/library/std/Cargo.toml
--- a/rust-src/lib/rustlib/src/rust/library/std/Cargo.toml
+++ b/rust-src/lib/rustlib/src/rust/library/std/Cargo.toml
@@ -64,7 +64,7 @@
 rand = { version = "0.9.0", default-features = false, features = ["alloc"] }
 rand_xorshift = "0.4.0"
 
-[target.'cfg(any(all(target_family = "wasm", target_os = "unknown"), target_os = "xous", all(target_vendor = "fortanix", target_env = "sgx")))'.dependencies]
+[target.'cfg(any(all(target_family = "wasm", any(target_os = "unknown", target_os = "none")), target_os = "xous", all(target_vendor = "fortanix", target_env = "sgx")))'.dependencies]
 dlmalloc = { version = "0.2.4", features = ['rustc-dep-of-std'] }
 
 [target.x86_64-fortanix-unknown-sgx.dependencies]
